metrics/cli/lib/gcloud/strings/gcloud_strings.dart

```diff
@@ -17,4 +17,16 @@ If you have already accepted the terms of the GCloud service, skip this step oth
 4. Accept the GCloud terms of service.
 5. Press a 'AGREE AND CONTINUE' button.
 Once you are done, press any key to continue:''';
+
+  static String configureOAuth(String projectId) => '''
+Use the following link for the correct work of the Metrics project: https://$projectId.firebaseapp.com. 
+
+If you are going to use any other domains to run the Metrics project, you should configure the OAuth 2.0 for them by following the steps listed below:
+
+1. Follow the link https://console.cloud.google.com/apis/credentials?project=$projectId/
+2. Press the 'Web client (auto created by Google Service)' name in the 'OAuth 2.0 Client IDs' section.
+3. Press the 'ADD URI' button at the end of the 'Authorized JavaScript origins' section.
+4. Enter the required domain.
```

Comment:
```suggestion
4. Enter the domain you want to use to access the Metrics Web application.
```
Would it be a bit more clear?